server/homebrew.api.js

```diff
@@ -89,148 +136,151 @@ const newBrew = async (req, res)=>{
 
 	beforeNewSave(req.account, brew);
 
-	let saved;
+	const newHomebrew = new HomebrewModel(brew);
+	newHomebrew.editId = nanoid(12);
+	newHomebrew.shareId = nanoid(12);
+
+	let googleId, saved;
```

Comment:
Another good place for a comment. Since `transferToGoogle` in this case actually means "save to google" rather than transferring out of the HB database.

1) Save Brew to Google Drive and prep a Stub
2) Save Brew (or Stub) to HB database

Reply:
I mean I can always change the query param name too

Reply:
transferToGoogle made sense before, but saveToGoogle makes more sense now.

Reply:
But still add comments :)

Reply:
Done!